CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+
+# 1.1.0
+
+Added `Test-CTypeDataMember` and `Add-CTypeData` functions for testing if a type has any defined custom type data and
+adding custom type data, respectively. Defining type data with .ps1xml files can result in errors importing the same
+module multiple times: PowerShell complains that the type data is already defined. Using `Add-CTypeData` prevents these
+this error as it only adds members that don't already exist.
+
+
+# 1.0.0
+
+## Upgrade Instructions
+
+We're breaking up Carbon into smaller and more targeted modules. Hopefully, this will help maintenance, and make it
+easier to use Carbon across many versions and editions of PowerShell. This module will be the place where core functions
+used by other Carbon modules will be put.
+
+If you're upgrading from Carbon to this module, you should do the following:
+
+* Replace usages of `Test-COSIs64Bit` with `Test-COperatingSystem -Is64Bit`.
+* Replace usages of `Test-COSIs32Bit` with `Test-COperatingSystem -Is32Bit`.
+* Replace usages of `Test-CPowerShellIs32Bit` with `Test-CPowerShell -Is32Bit`.
+* Replace usages of `Test-CPowerShellIs64Bit` with `Test-CPowerShell -Is64Bit`.
+* Rename usages of the `ConvertTo-CBase64` function's `Value` parameter to `InputObject`, or pipe the value to the 
+function instead.
+
+We made a lot of changes to the `Invoke-CPowerShell` function:
+
+* The `Invoke-CPowerShell` function no longer allows you to pass script blocks. Instead, convert your script block to
+a string, and pass that string to the `Command` parameter. This will base64 encode the command and pass it to 
+PowerShell's -EncodedCommand property.
+* The `Invoke-CPowerShell` function no longer has `FilePath`, `OutputFormat`, `ExecutionPolicy`, `NonInteractive`, 
+or `Runtime` parameters. Instead, pass these as arguments to the `ArgumentList` parameter, e.g. 
+`-ArgumentList @('-NonInteractive', '-ExecutionPolicy', 'Bypasss'). You are now responsible for passing all PowerShell

```

Comment:
Markdown formatting isn't right on the `-ArgumentList` example because you're missing a backtick between the parenthesis and the period in `Bypass').`

Reply:
Fixed.